core/cat/routes/memory.py

```diff
@@ -151,42 +137,57 @@ async def wipe_single_collection(request: Request, collection_id: str = "") -> D
     }
 
 
-# DELETE all collections
-@router.delete("/wipe-collections/")
-async def wipe_collections(
+# DELETE conversation history from working memory
+@router.delete("/conversation_history/")
+async def wipe_conversation_history(
     request: Request,
 ) -> Dict:
-    """Delete and create all collections"""
+    """Delete conversation history from working memory"""
 
     ccat = request.app.state.ccat
-    collections = list(ccat.memory.vectors.collections.keys())
-    vector_memory = ccat.memory.vectors
-
-    to_return = {}
-    for c in collections:
-        ret = vector_memory.vector_db.delete_collection(collection_name=c)
-        to_return[c] = ret
-
-    ccat.load_memory()  # recreate the long term memories
-    ccat.mad_hatter.find_plugins()
-    ccat.mad_hatter.embed_tools()
+    ccat.working_memory["history"] = []
 
     return {
         "status": "success",
-        "deleted": to_return,
+        "deleted": True,
     }
 
-#DELETE conversation history from working memory
-@router.delete("/working-memory/conversation-history/")
-async def wipe_conversation_history(
+
+# DELETE memories
+@router.delete("/{collection_id}/point/{memory_id}/")
```

Comment:
@pieroit would you prefer like that or directly `/{collection_id}/{memory_id}/` ?

Reply:
let's leave `point`, in the future we may consider other aspects of a collection as a resource

Reply:
Ok, perfect!

Reply:
thanks for fixing the tests!!!
Going for `/collections/{collection_id}/points/{memory_id}/` which should be the most RESTful